my-app/src/SearchPage.tsx

```diff
@@ -13,50 +13,57 @@ export interface SearchResult {
 }
 
 function SearchPage() {
-  const [searchQuery, setSearchQuery] = useState<string>('');
+  const [firstName, setFirstName] = useState<string>('');
+  const [lastName, setLastName] = useState<string>('');
+  const [cardNumber, setCardNumber] = useState<string>('');
   const [searchResults, setSearchResults] = useState<SearchResult[]>([]);
 
-
   const submitForm = async (): Promise<void> => {
     try {
-      const response = await fetch('https://api.afg.com/account-search', {
+      const response = await fetch('https://api.afg.com/cardholders/search', {
         method: 'POST',
         headers: {
           'Content-Type': 'application/json',
         },
-        body: JSON.stringify(searchQuery),
+        body: JSON.stringify({ firstName, lastName, cardNumber }),
       });
 
       if (!response.ok) {
         throw new Error(`HTTP error! Status: ${response.status}`);
       }
 
       const results: SearchResult[] = await response.json();
-      setSearchResults(results)
+      setSearchResults(results);
     } catch (error) {
       console.error('Error fetching search results:', error);
-      throw error;
     }
   };
 
-
   const handleSearch = (e: FormEvent<HTMLFormElement>) => {
     e.preventDefault();
-    submitForm()
-  };
-
-  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
-    setSearchQuery(e.target.value);
+    submitForm();
   };
 
   return (
     <div>
```

Comment:
Can we talk to the design team about adding a success / error alert to show the result of the request as visual feedback?